internal/topology/variables/schema_test.go

```diff
@@ -49,6 +49,24 @@ func Test_convertToAPIExtensionsJSONSchemaProps(t *testing.T) {
 				ExclusiveMaximum: true,
 				Minimum:          ptr.To[int64](1),
 				ExclusiveMinimum: false,
```

Comment:
If this is only allowed for object, I would prefer to only add it to the object unit test here (vs. the noise of adding it in places where it's not valid anyway)

Reply:
This is not only applicable to objects so leaving here.